src/v3/src/components/LaunchAuthenticatorButton/LaunchAuthenticatorButton.tsx

```diff
@@ -39,13 +40,24 @@ const LaunchAuthenticatorButton: UISchemaElementComponent<{
   } = uischema;
 
   const focusRef = useAutoFocus<HTMLButtonElement>(focus);
+  const {
+    userIdentifier,
+    setUserIdentifier,
+    data,
+    widgetProps: { features },
+  } = useWidgetContext();
 
   const handleClick: ClickHandler = async () => {
+    if (features?.engFastpassMultipleAccounts && data.identifier) {
```

Comment:
```suggestion
    if (data.identifier) {
```

Reply:
Per latest comment in Jira, we are removing this flag check

Reply:
https://github.com/okta/okta-signin-widget/pull/3034

Reply:
Let's remove the check here and update the tests in the same fashion as in #3034. It will cause some conflicts on rebase/merge to the trunk but they shouldn't be difficult to resolve.

Reply:
👍